storage-providers/filecoin-deals/filecoin-programs.md

```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/go-swan-provider/README.md)
+
+A golang implementation for Swan Service provider.
+
+#### Multichain Storage
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/multi-chain-storage)](https://github.com/filswan/multi-chain-storage)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/multi-chain-storage)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/multi-chain-storage/README.md)
+
+[Multichain Storage](https://multichain.storage/home) is a smart-contract-based cross-chain storage gateway integrated with Oracle technology and the Filecoin networks. It accelerates the mass adoption of decentralized storage by bridging multiple blockchain networks.
+
+#### FS3
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/fs3)](https://github.com/filswan/fs3)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/fs3)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/fs3/README.md)
+
+A high Performance, Kubernetes Native Object Storage.
+
+## Deprecated
+
+Here is a comprehensive list of deprecated tools and projects.
+
+## Filecoin Tracker
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated_04/20/2024-red.svg)
+
+[Filecoin Tracker](https://filecoin-tracker.com/) was deprecated on April 20, 2024.
+
+Here are great existing and working Filecoin dashboards that cover similar topics:
+
+- [Starboard](https://dashboard.starboard.ventures/dashboard)
+- [Filecoin Dune Daily Metrics](https://dune.com/kalen/filecoin-daily-metrics)
+- [Filecoin Pulse (PoC)](https://filecoinpulse.pages.dev/)
+
+## Slingshot
+
+[![GitHub Repo stars](https://img.shields.io/github/stars/filecoin-project/slingshot)](https://github.com/filecoin-project/slingshot)
+![GitHub last commit](https://img.shields.io/github/last-commit/filecoin-project/slingshot)
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C01AZP8BKRQ)
+
+[Slingshot](https://slingshot.filecoin.io) was a program that united Data clients, Data preparers and storage providers in a community to onboard data and share replicas of publicly valuable [_Open Datasets_](https://datasets.filecoin.io). Rather than providing a web interface like Estuary, Slingshot was a program that provides a workflow and tools for onboarding of large open datasets. The Slingshot Deal Engine provided deals to registered and certified storage providers. The data was prepared and uploaded using a tool called Singularity. The program has [clear requirements of a storage provider](https://slingshot.filecoin.io/requirements#participating-as-a-storage-provider-sp) regarding their capabilities.
```

Comment:
This is good.
link 'Singularity' to the singularity github repo. and remove the last sentence.